inginious/frontend/user_manager.py

```diff
@@ -417,6 +417,11 @@ def get_user_api_key(self, username, create=True):
             apikey = retval.get("apikey", None)
         return apikey
 
+    def get_course_user_settings(self, username, courseid):
```

Comment:
Do not forget default values has to be passed in case they are not saved (yet) in DB.